web/css/src/themes/default.css

```diff
@@ -2438,9 +2439,8 @@
 }
 
 .server-console-output {
-	color: #c36;
-	padding: 10px 0 20px 20px;
 	background: #fff;
```

Comment:
```suggestion
	background-color: #fff;
```

Reply:
I know it's not part of your changes but would also be good to change `background` to `background-color` in the same place in `dark.css`. Thanks!